sds_data_manager/utils/stackbuilder.py

```diff
@@ -19,7 +21,13 @@
 
 
 def build_sds(
-    scope: App, env: Environment, sds_id: str, use_custom_domain: bool = False
+        scope: App,
+        env: Environment,
+        sds_id: str,
+        rds_size: str = 'SMALL',
```

Comment:
I'm concerned with how many arguments this function has. Can you think of a good way to reduce the number of arguments in here? Maybe by putting all the database settings into a config class?

Reply:
I do agree here. Those new argument are RDS specific and it make sense to refactor it out somewhere else.

Reply:
:+1: to the above. I guess the question is "how many of these knobs do we actually want to mess with?" versus just hard-coding them and putting in comments about why you chose something. If we do want the knobs, I think we could put them in the `cdk.json` configuration area possibly so we could choose different settings for each account. But, I'm not sure how many of these we really need versus just selecting a value and going with it.